downstream/modules/platform/con-edge-manager-agent-service.adoc

```diff
@@ -4,7 +4,7 @@
 
 // The intro mentions ACM, confirm if different for AAP
 
-The {RedHatEdge} agent is a process running on each managed device that periodically communicates the {RedHatEdge} service on the ACM hub cluster.
+The {RedHatEdge} agent is a process running on each managed device that periodically communicates the {RedHatEdge} service.
```

Comment:
Is there a word missing? 
```suggestion
The {RedHatEdge} agent is a process running on each managed device that periodically communicates with the {RedHatEdge} service.
```
I might be reading it incorrectly. Ignore if this was intended.